source/includes/write/transaction.kt

```diff
@@ -1,6 +1,5 @@
-import com.mongodb.*
-import com.mongodb.client.ClientSession
-import com.mongodb.client.MongoClients
+import com.mongodb.kotlin.client.ClientSession
+import com.mongodb.kotlin.client.MongoClient
```

Comment:
[nit] I see that imports are not displayed on the page. However,  If there is ever a decision to include them, then It may be necessary to include the following imports to ensure the code compiles correctly on user/reader machines:
```
import com.mongodb.ReadConcern
import com.mongodb.TransactionOptions
import com.mongodb.WriteConcern
```

Just to not forget about them in the future, i suggest adding those.

Reply:
Thanks for this feedback, will add those imports in case they're ever displayed or if another writer is referring to this example 👍